Signal/src/ViewControllers/ConversationView/ConversationViewItem.h

```diff
@@ -55,10 +55,9 @@ NSString *NSStringForOWSMessageCellType(OWSMessageCellType cellType);
 @property (nonatomic, readonly) BOOL hasQuotedText;
 
 @property (nonatomic) BOOL shouldShowDate;
-// TODO: Consider renaming to shouldHideFooter.
-@property (nonatomic) BOOL shouldHideRecipientStatus;
-// Used to suppress "group sender" avatars.
-@property (nonatomic) BOOL shouldHideAvatar;
+@property (nonatomic) BOOL shouldShowSenderAvatar;
```

Comment:
It seems a little strange that some of these BOOL's are phrased as Show and others as Hide, but no biggie.

Reply:
The intention was to frame these in such a way that the common case was NO.